release_images.yml

```diff
@@ -2,182 +2,45 @@
-    version: "2.9.1"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  7:
-    framework: "tensorflow"
-    version: "2.9.1"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: True
-      disable_sm_tag: True  # [Default: False] This option is not used by Example images
-      force_release: False
-  8:
-    framework: "huggingface_tensorflow_trcomp"
-    version: "2.6.3"
-    hf_transformers: "4.17.0"
-    arch_type: "x86"
-    training:
-      device_types: [ "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
-  9:
     framework: "pytorch"
     version: "1.11.0"
     arch_type: "x86"
     customer_type: "sagemaker"
     training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      cuda_version: "cu113"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: False
-      force_release: False
-  10:
-    framework: "pytorch"
-    version: "1.12.0"
-    arch_type: "x86"
-    customer_type: "sagemaker"
-    training:
-      device_types: [ "cpu", "gpu" ]
+      device_types: ["gpu"]
       python_versions: [ "py38" ]
       os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
-  11:
-    framework: "huggingface_pytorch_trcomp"
-    version: "1.10.2"
-    hf_transformers: "4.17.0"
-    arch_type: "x86"
-    training:
-      device_types: [ "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
-  12:
-    framework: "autogluon"
-    version: "0.5.2"
-    arch_type: "x86"
-    training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      cuda_version: "cu115"
```

Comment:
For SM, did we release cu113 earlier or cu115?

Reply:
Changing this to cu113.